runtime/server_http_request.go

```diff
@@ -451,3 +477,8 @@ func (req *ServerHTTPRequest) UnmarshalBody(
 
 	return true
 }
+
+// GetContext returns the http request Context
+func (req *ServerHTTPRequest) GetContext() context.Context {
```

Comment:
This one liner function seems too simple to be necessary, the only call site is https://github.com/uber/zanzibar/pull/354/files#diff-2fd36df947abf02141a9e4d07dd5367dR834

Reply:
needed a way to check span out of package scope in test.  let me know if you have a way to improve

Reply:
Oh, I see, didn't notice it was in a different package. Makes sense.